packages/web/src/components/gcds-select/gcds-select.tsx

```diff
@@ -90,14 +92,40 @@ export class GcdsSelect {
    */
   @Prop({ reflect: true, mutable: false }) defaultValue?: string;
 
+
+  /**
+   * If true, the select will be focused on component render
+   */
+  @Prop({ reflect: true }) autofocus: boolean;
```

Comment:
Should this be optional too? And/or have an inferred default on declaration?
```suggestion
  @Prop({ reflect: true }) autofocus?: false;
```